### ISSUE TITLE
Version 8.1.0?

### ISSUE THREAD

**Issue by smk508** [cptactionhank/docker-atlassian-jira-software#61]
I noticed that version 8.1.0 is no longer available, and also the latest tag now points to an older version of Jira. Is there any way to get the 8.1.0 image back?

**Comment by bobswift** [comment]
I noticed the CI builder says: Updated Atlassian JIRA Software master branch to latest version 8.1.0. So, it would seem CI builder knows about 8.1.0 from Atlassian at least.

**Comment by perlan** [comment]
Looks like JIRA_VERSION was updated but Dockerfile is still fetching atlassian-jira-software-7.13.3.tar.gz. This seems a little brittle and would probably be fixed by PR #56.

**Comment by smk508** [comment]
Also the CircleCI builder keeps reverting master to 7.13 after trying to upgrade to 8.1. It just did it again yesterday.

**Comment by perlan** [comment]
Yes, It seems like the bin/prepare script is not working correctly. It will always pick the last entry for the tar file (the opposite to the comment in the file) and update the JIRA_VERSION back and forth between the two available versions. My guess is that previously, it has been only one entry in the Atlassian response.

**Comment by perlan** [comment]
The version feed is now back to just single jira version (8.1.0) so the latest build is back to using 8.1.0. The problem in bin/prepare is still there so it will break again the next time there are multiple versions in the version feed.

**Comment by smk508** [comment]
I think that atlassian considers 7.13.3 to be the 'long-term' version of Jira while 8.1 is the latest one, which is why the feed gives tars for both versions. So perhaps both of them should be built into separate branches?
Also, even though 8.1 is on master now, there isn't a separate branch for it like for the other Jira versions, so there's no way to pin to that version when pulling from Docker other than to pull latest.

Also, this is the line in the bash script that pulls the version feed and is causing issues for reference:
https://github.com/cptactionhank/docker-atlassian-jira-software/blob/e2a08c4e8608603b281f8bb441e7614436dda499/bin/prepare#L25

**Comment by zoispag** [comment]
It is now done.

**Comment by blop** [comment]
https://hub.docker.com/r/cptactionhank/atlassian-jira/tags
https://hub.docker.com/r/cptactionhank/atlassian-jira-software/tags

Still no sign of the 8.1+

**Comment by zoispag** [comment]
@blop maybe not explicitly tagged, but it is there.

![image](https://user-images.githubusercontent.com/21138205/57455354-41a65e00-726b-11e9-8d36-daab77e9e3e0.png)

I am using latest: `cptactionhank/atlassian-jira-software:latest`

**Comment by blop** [comment]
Using ":latest" in production configurations is not recommended : 
https://vsupalov.com/docker-latest-tag/

**Comment by viceice** [comment]
> https://hub.docker.com/r/cptactionhank/atlassian-jira/tags
> https://hub.docker.com/r/cptactionhank/atlassian-jira-software/tags
> 
> Still no sign of the 8.1+

at least jira software 8.1.0 is finally there

**Comment by cptactionhank** [comment]
As long as the feed (https://my.atlassian.com/download/feeds/current/jira-software.json) does not contain any later version, this will not have that either. 

You could push Atlassian for updating their feed.

**Comment by blop** [comment]
https://my.atlassian.com/download/feeds/current/jira-software.json -> JIRA 8.2.1
cptactionhank/atlassian-jira-software:8.2.1 not found: manifest unknown: manifest unknown

Maybe it's the same issue ?

**Comment by loneken79** [comment]
As a workaround you can pull the latest images and tag it localy, so you can use it in your process.

Assuming the latest commit is refering to the desired tag update (8.3.0 actually) the commands are:
1. `docker pull cptactionhank/atlassian-jira-software` (get "latest" version of the image)
2. `docker tag cptactionhank/atlassian-jira:latest cptactionhank/atlassian-jira:8.3.0` (tag this image locally)
3. check that you see both "latest" and "8.3.0" version with the same "Image ID" in `docker images | head` listing
4. `docker rmi cptactionhank/atlassian-jira:latest` (clean unused "latest" version)
